Packs/Elasticsearch/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Elasticsearch v2
+- Fixed an issue where JSON incident labels were not saved properly.
```

Comment:
i would elaborate on the change in rn as this theoretically can break bc

Reply:
as most of our bugs, we used to have bad behavior, and now it was fixed.
simple object behavior did not change only complex objects.